.pre-commit-config.yaml

```diff
@@ -24,7 +24,7 @@ repos:
         additional_dependencies: ["@commitlint/config-angular"]
         args: ["--config", "./commitlint.config.js"]
   - repo: https://github.com/macisamuele/language-formatters-pre-commit-hooks
-    rev: v2.10.0
+    rev: v2.12.0
```

Comment:
What does this bump fix for you?

Reply:
It fixed missing Python `setuptools` for me.
As it was only a local problem, I did not mention it in the PR.

https://github.com/macisamuele/language-formatters-pre-commit-hooks/commit/e43d17e9db27aebd8d73dff9c2f353a11204bc8e

Reply:
Ah, I see. I'd be more in favor of putting such things into separate PRs to avoid polluting the interesting content discussions and diffs with such issues.

Reply:
If it helps, I'll do it that way in the future  😄